src/olympia/activity/utils.py

```diff
@@ -227,33 +227,47 @@ def log_and_notify(action, comments, note_creator, version, perm_setting=None,
         subject = u'Mozilla Add-ons: %s %s %s' % (
             version.addon.name, version.version, action.short)
     template = template_from_user(note_creator, version)
-    send_activity_mail(
-        subject, template.render(Context(author_context_dict)), version,
-        addon_authors, settings.EDITORS_EMAIL, perm_setting)
-    send_activity_mail(
-        subject, template.render(Context(reviewer_context_dict)), version,
-        reviewers, settings.EDITORS_EMAIL, perm_setting)
+
```

Comment:
You don't need all this change - just do what I did already with `reviewer_context_dict` and create a new `_context_dict` for in staff_cc (you'll need to do some set logic to get a list of cc users who aren't in reviewers, addon_authors, task-user and the note_creator).  Then have a `third send_activity_mail`